### ISSUE TITLE
feature=history has new accessibility issues

### ISSUE THREAD

**Issue by steren** [electricitymap/electricitymap-contrib#4301]
The new "history" feature introduces new controls:
![image](https://user-images.githubusercontent.com/360895/176189269-f5e436e4-3518-491c-8ba2-53a5d79e9545.png)
![image](https://user-images.githubusercontent.com/360895/176189315-39fd52a6-d750-4d13-ade4-2c2c4f41b11f.png)

 
A Lighthouse test reveals that they suffer from accessibility issues. This should be considered a regression.

It should be easily fixable by following [this guidance](https://web.dev/button-name/?utm_source=lighthouse&utm_medium=devtools) and add new `aria-label` attributes describing the action of the button

![image](https://user-images.githubusercontent.com/360895/176189421-5289a667-386b-45ff-8444-2d0d9cb8cf59.png)

**Comment by madsnedergaard** [comment]
Thanks for raising this @steren! 🙌 

We haven't really looked at accessibility yet, but it's a good point and reminder - I'll run Lighthouse and solve the issues :)

Minor side-note, but it became too hard to keep up with the feature-flag, so we're currently working on [a separate `historical-view-v1` branch](https://github.com/electricitymap/electricitymap-contrib/tree/historical-view-v1) instead of releasing latest changes to production :)

**Comment by VIKTORVAV99** [comment]
> Thanks for raising this @steren! 🙌 
> 
> We haven't really looked at accessibility yet, but it's a good point and reminder - I'll run Lighthouse and solve the issues :)
> 
> Minor side-note, but it became too hard to keep up with the feature-flag, so we're currently working on [a separate `historical-view-v1` branch](https://github.com/electricitymap/electricitymap-contrib/tree/historical-view-v1) instead of releasing latest changes to production :)

Lighthouse is great but I find it cumbersome for active development and would recommend complementing it with this: [webhint](https://webhint.io/)

They have a nice vscode extension that hint and lint the source files directly while working on them so you can avoid the countless lighthouse runs.
